examples/README.md

```diff
@@ -1,8 +1,8 @@
 # NVIDIA FLARE Examples
 
-[NVIDIA FLARE](https://nvflare.readthedocs.io/en/2.1.1/index.html) provides several examples to help you get started using federated learning for your own applications.
+[NVIDIA FLARE](https://nvflare.readthedocs.io/en/main/index.html) provides several examples to help you get started using federated learning for your own applications.
 
-The provided examples cover different aspects of [NVIDIA FLARE](https://nvflare.readthedocs.io/en/2.1.1/index.html), such as using the provided [Controllers](https://nvflare.readthedocs.io/en/2.1.1/programming_guide/controllers.html) for "scatter and gather" or "cyclic weight transfer" workflows and example [Executors](https://nvflare.readthedocs.io/en/2.1.1/apidocs/nvflare.apis.html?#module-nvflare.apis.executor) to implement your own training and validation pipelines. Some examples use the provided "task data" and "task result" [Filters](https://nvflare.readthedocs.io/en/2.1.1/apidocs/nvflare.apis.html?#module-nvflare.apis.filter) for homomorphic encryption and decryption or differential privacy. Furthermore, we show how to use different components for FL algorithms such as [FedAvg](https://arxiv.org/abs/1602.05629), [FedProx](https://arxiv.org/abs/1812.06127), and [FedOpt](https://arxiv.org/abs/2003.00295). We also provide domain-specific examples for deep learning and medical image analysis.
+The provided examples cover different aspects of [NVIDIA FLARE](https://nvflare.readthedocs.io/en/main/index.html), such as using the provided [Controllers](https://nvflare.readthedocs.io/en/main/programming_guide/controllers.html) for "scatter and gather" or "cyclic weight transfer" workflows and example [Executors](https://nvflare.readthedocs.io/en/dev-2.1/apidocs/nvflare.apis.executor.html) to implement your own training and validation pipelines. Some examples use the provided "task data" and "task result" [Filters](https://nvflare.readthedocs.io/en/main/apidocs/nvflare.apis.html?#module-nvflare.apis.filter) for homomorphic encryption and decryption or differential privacy. Furthermore, we show how to use different components for FL algorithms such as [FedAvg](https://arxiv.org/abs/1602.05629), [FedProx](https://arxiv.org/abs/1812.06127), and [FedOpt](https://arxiv.org/abs/2003.00295). We also provide domain-specific examples for deep learning and medical image analysis.
```

Comment:
I was using dev-2.1 to find the correct links for the apidocs pages since readthedocs built them slightly differently than the script on github.io, but only replaced 2.1.1 -> main. Thanks for spotting this, I will update these.